### PR TITLE
Added path replacer xml

### LINKED ISSUE #1371: Add more language specific path replacers
I've prepared an XML with different path replacers in different languages.

### LINKED ISSUE #1548: Problem with PathCleanup and special characters (é, á)
We've tried to copy a locale to another and it stopped at a page which had `é` in it's title. The path cleaner can't handle those characters because in the `cleanup` function, **strtolower** returned something like `%A0`.

Instead of this...
````
    $clean = strtolower($dirty);

    $replacers = array_merge(
        $this->replacers['default'],
        (isset($this->replacers[$languageCode]) ? $this->replacers[$languageCode] : [])
    );

    if (count($replacers) > 0) {
        foreach ($replacers as $key => $value) {
            $clean = str_replace($key, $value, $clean);
       }
    }

```

... the **strtolower** should happen afterwards to fix this bug.

$replacers should also be filled with more characters, for example:
````

            'á' => 'a',
            'Á' => 'a',
            'à' => 'a',
            'À' => 'a',
            'é' => 'e',
            'É' => 'e',
            'è' => 'e',
            'È' => 'e',
```

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1371 , fixes #1548
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
I didn't see a single test succeed :stuck_out_tongue:

File: src/Sulu/Component/PHPCR/PathCleanup.php [hunk 1]
```diff
@@ -26,17 +26,15 @@ class PathCleanup implements PathCleanupInterface
...
-            // because strtolower ignores Ä,Ö,Ü
             'Ä' => 'ae',
             'Ö' => 'oe',
             'Ü' => 'ue',
             'ß' => 'ss',
-            // TODO should be filled
-        ],
-        'de' => [
             '&' => 'und',
         ],
```
Comment (danrot) [inline src/Sulu/Component/PHPCR/PathCleanup.php]: If we read that from an xml file, these lines should be deleted.

**Comment by danrot** [comment]
@wachterjohannes Now at least a few less tests are failing ;-)

**Comment by wachterjohannes** [comment]
@danrot i think this should work now (:

**Comment by danrot** [comment]
Spaces are now replaced by `+` instead of `-`, and the javascript component say this is not valid, so the page cannot be saved.

@crash0verride Should spaces in URLs be replaced by a `+` or a `-`?

**Comment by wachterjohannes** [comment]
the space is completelly missing in the xml file, i will add it with `-` as replacer because this was the behavior before

**Comment by crash0verride** [comment]
@danrot @wachterjohannes I would suggest the - as this is common practice.

**Pull request merged**